<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assets</class>
 <widget class="QMainWindow" name="Assets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>733</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Implicit Terrain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_GL" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="Parameters_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>141</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string> Statistics</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertex</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Triangles</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="resetcameraButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;Callback #03&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#aa00ff;&quot;&gt;Create Eurographics 1996 Bird&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Reset Camera</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>251</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Shading</string>
       </property>
       <widget class="QRadioButton" name="radioShadingButton_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="wireframe">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioShadingButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="bezier">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>311</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Bezier example</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="revolution">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>311</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Revolution example</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Implicit examples</string>
       </property>
       <widget class="QPushButton" name="implicit_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Replication (Box - Torus)</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="implicit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Box - Torus</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="implicit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Table (Blend Box / Capsules)</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="implicit_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Intersection Sphere / Boite</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionExport_obj">
   <property name="text">
    <string>OBJ File (.obj)</string>
   </property>
  </action>
  <action name="actionExport_ma">
   <property name="text">
    <string>Maya File (.ma)</string>
   </property>
  </action>
  <action name="actionLoad_ma">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionGL">
   <property name="text">
    <string>GL</string>
   </property>
  </action>
  <action name="actionVectorized">
   <property name="text">
    <string>Vectorized</string>
   </property>
  </action>
  <action name="actionExport_svg">
   <property name="text">
    <string>SVG File (.svg)</string>
   </property>
  </action>
  <action name="actionSphere_Tracing_png">
   <property name="text">
    <string>Sphere Tracing (.png)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
